shuffle/prisma/schema.prisma

```diff
@@ -29,11 +29,14 @@ model Statement {
 model FlaggedStatement {
   id          Int       @id @unique @default(autoincrement())
   statementId Int
+  poll_id     Int
   user_id     String?
   session_id  String    @db.VarChar
   reason      String?
   created_at  DateTime  @default(now()) @db.Timestamptz(6)
   statement   Statement @relation(fields: [statementId], references: [id], onDelete: Cascade, onUpdate: NoAction)
+  poll        polls     @relation(fields: [poll_id], references: [id], onDelete: Cascade, onUpdate: NoAction)
```

Comment:
Why do we need this? This adds redundancy because statement already has this data.